.github/workflows/android-ci.yml

```
@@ -0,0 +1,47 @@
+                    distribution: 'corretto'
+                    java-version: 17
+
+            -   name: Setup Android SDK
+                uses: android-actions/setup-android@v2
+
+            -   name: Setup Gradle
+                uses: gradle/gradle-build-action@v2
+                with:
+                    gradle-home-cache-cleanup: true
+
+            -   name: Generate local.properties
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)